includes/class-wcpdf-admin.php

```diff
@@ -1194,6 +1196,14 @@ public function process_order_document_form_data( $form_data, $document_slug )
 
 		return $data;
 	}
+
+	public function add_invoice_number_to_order_report( $response ) {
+		if ( ! empty( $order = wc_get_order( $response->data['order_id'] ) ) ) {
```

Comment:
I think the variable assignment should be in one separated line to comply with WPCS.

Reply:
Yeah, thanks for noticing.